### DISCUSSION TITLE
Working with multiselect

### DISCUSSION THREAD

**Discussion by reinisg** [react-hook-form/react-hook-form#4294] [Q&A] [answered]
**Describe the bug**
Main problem is if I add/ remove item from multiselect not showing isDirty to true. And when press 'Submit' button, submitted data is not changed. I think I have missed some settings, can someone help with this issue?

**To Reproduce**
1. Go to [sandbox example](https://codesandbox.io/s/sweet-pasteur-l3xix?file=/src/App.js)
2. Add/ remove item form multiselect
3. Press submit button

**Expected behavior**
On submit in console submitted data shows correct selected data and isDirty true - because the field has changed

**Additional context**
 "react": "17.0.1",
"multiselect-react-dropdown": "^1.6.10",
"react-hook-form": "^6.15.1"

Hope for some code advices, 
Regards Reinis

**Comment by bluebill1049** [comment, marked as answer]
<img width="661" alt="Screen Shot 2021-02-27 at 10 04 40 pm" src="https://user-images.githubusercontent.com/10513364/109385419-ca622280-7947-11eb-87f3-740a39ff68de.png">
you are not using any of the props from render.

have a read at https://react-hook-form.com/api#Controller:

also there is a CSB: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r check out the autocomplete example.
